src/Readm/Readm.ts

```diff
@@ -0,0 +1,194 @@
+  TagType,
+  TagSection,
+  MangaTile
+} from "paperback-extensions-common"
+import { parseUpdatedManga, generateSearch, parseChapterDetails, isLastPage, parseTags, parseChapters, parseHomeSections, parseMangaDetails, parseViewMore, UpdatedManga } from "./ReadmParser"
+
+const RM_DOMAIN = 'https://readm.org'
+
+const method = 'GET'
+
+export const ReadmInfo: SourceInfo = {
+  version: '1.0.0',
+  name: 'Readm',
+  icon: 'icon.png',
+  author: 'Netsky',
+  authorWebsite: 'https://github.com/TheNetsky',
+  description: 'Extension that pulls manga from Readm.',
+  hentaiSource: false,
+  websiteBaseURL: RM_DOMAIN,
+  sourceTags: [
+    {
+      text: "Notifications",
+      type: TagType.GREEN
+    }
+  ]
+}
+
+export class Readm extends Source {
+  getMangaShareUrl(mangaId: string): string | null { return `${RM_DOMAIN}/manga/${mangaId}` }
+
+  async getMangaDetails(mangaId: string): Promise<Manga> {
+    const request = createRequestObject({
+      url: `${RM_DOMAIN}/manga/`,
+      method,
+      param: mangaId,
+    })
+
+    const response = await this.requestManager.schedule(request, 1);
+    const $ = this.cheerio.load(response.data);
+    return parseMangaDetails($, mangaId);
+  }
+
+  async getChapters(mangaId: string): Promise<Chapter[]> {
+    const request = createRequestObject({
+      url: `${RM_DOMAIN}/manga/`,
+      method,
+      param: mangaId,
+    })
+
+    const response = await this.requestManager.schedule(request, 1);
+    const $ = this.cheerio.load(response.data);
+    return parseChapters($, mangaId);
+  }
+
+  async getChapterDetails(mangaId: string, chapterId: string): Promise<ChapterDetails> {
+    const request = createRequestObject({
+      url: `${RM_DOMAIN}/manga/${mangaId}/${chapterId}`,
+      method: "GET",
+    });
+
+    this.getTags()
+    const response = await this.requestManager.schedule(request, 1);
+    const $ = this.cheerio.load(response.data, { xmlMode: false });
+    return parseChapterDetails($, mangaId, chapterId);
+  }
+
+  async getTags(): Promise<TagSection[] | null> {
+    const request = createRequestObject({
+      url: `${RM_DOMAIN}`,
+      method,
+    })
+
+    const response = await this.requestManager.schedule(request, 1);
+    const $ = this.cheerio.load(response.data);
+    return parseTags($);
+  }
+
+  async filterUpdatedManga(mangaUpdatesFoundCallback: (updates: MangaUpdates) => void, time: Date, ids: string[]): Promise<void> {
+    let page = 1;
+    let updatedManga: UpdatedManga = {
+      ids: [],
+      loadMore: true
+    };
+
+    while (updatedManga.loadMore) {
+      const request = createRequestObject({
+        url: `${RM_DOMAIN}/latest-releases/${page++}`,
+        method,
+      })
+
+      const response = await this.requestManager.schedule(request, 1)
+      const $ = this.cheerio.load(response.data)
+
+      updatedManga = parseUpdatedManga($, time, ids)
+      if (updatedManga.ids.length > 0) {
+        mangaUpdatesFoundCallback({
+          ids: updatedManga.ids
```

Comment:
createMangaUpdates() needed here
```suggestion
          createMangaUpdates({ ids: updatedManga.ids });
```